tools/gsc-mannwhitney_de/mannwhitney_de.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="mannwhitney_de" name="Perform a differential analysis" version="0.1.0">
+    <description>between the two groups of a signature score using a Mann-Whitney test</description>
+    <requirements>
+        <requirement type="package" version="1.3.2=r3.3.2_0">r-optparse</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Tool exception" />
+    </stdio>
+    <command detect_errors="exit_code"><![CDATA[ 
+        Rscript $__tool_directory__/MannWhitney_DE.R 
+            --input $input 
+            --sep 
+            #if $sep == 'tab':
+              'tab'
+            #elif $sep == 'comma':
+              'comma'
+            #end if
+            --colnames $colnames
+            --metadata $metadata
+            --name $name
+            --fdr $fdr
+            --output $output
+]]></command>
+    <inputs>
+        <param name="input" type="data" format="txt" label="Expression data" help="a csv or tsv table file" />
+        <param name="sep" type="select" label="Indicate column separator">
+            <option value="tab" selected="true">Tabs</option>
+            <option value="comma">Comma</option>
+        </param>
+        <param name="colnames" type="select" label="Firt row contains column names">
+            <option value="TRUE" selected="true">Yes</option>
+            <option value="FALSE">No</option>
+        </param>
+        <param name="metadata" type="data" format="tabular" label="Cell metadata"
+               help="A tsv table file with at least two columns : cell barcodes and a column that split cells into two categories." />
```

Comment:
"A tsv table file with at least two columns : cell barcodes and a column that split cells into two categories `taking two different values of a factor`"

Reply:
Note that this is obviously note particularly a "metadata" file, just a file indicating the factor values taking by tested items

Reply:
`help="A tsv table file with at least two columns : cell barcodes and a column that split cells into two categories." />`

-->

`help="A tsv table file with two columns : a column with cell identifiers and a column that contains the value of the comparison factor for the cell." />`